src/main/java/ua/goit/gojava32/kickstarter/controller/AdminPageController.java

```diff
@@ -0,0 +1,36 @@
+package ua.goit.gojava32.kickstarter.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.ModelAndView;
+import ua.goit.gojava32.kickstarter.model.Category;
+import ua.goit.gojava32.kickstarter.service.CategoryService;
+import java.util.List;
+
+@Controller
+public class AdminPageController {
+
+  @Autowired
+  private CategoryService categoryService;
+
+  @RequestMapping(value = "/admin/edit_category", method = RequestMethod.GET)
```

Comment:
Google for: REST conventions. Edit request is usually mapped to POST Http method